managed/ui/src/components/config/Security/certificates/AddCertificateFormContainer.js

```diff
@@ -27,7 +27,7 @@ const mapDispatchToProps = (dispatch) => {
           });
         } else {
           console.error(
-            'Certificate adding has been failed: ' + JSON.stringify(response.payload.data.error)
+            'Certificate adding has been failed: ' + JSON.stringify(response?.payload?.data?.error)
```

Comment:
Nice to use toast message instead console . I know its already there , but please change as user not checking with console .

Reply:
Done